File: section4/testMovieGenre.py
from movieGenre import *

print(Genre)

movie = Movie()
thor = Movie()
thor.title = "Thor Ragnarok"

genre = Genre()
action = Genre()
action.name = "Action"

print(action.name)

genre.addMovie(movie)
 #print genre.movies
#print (genre['name'])

print (genre.movies)

print (movie.setGenre(genre))

print(thor.setGenre(action))
print(action.movies[0].title)

darkWorld = Movie()
darkWorld.title = "Dark World"
print(thor.addRelatedMovie(darkWorld))
print(thor.related[0].title)

File: section4/testMovieGenre.js
const movieGenre = require('./movieGenre').movieGenre;
const assert = require('assert');

function createMovie(){
    return new movie(1, "Buba", 1, 1, 3);
}

function test_construtor(){
    let p = createPokemon();
    assert.deepEqual(p.name, "Buba");
}



